ql/src/java/org/apache/hadoop/hive/ql/metadata/HiveStorageHandler.java

```diff
@@ -796,7 +796,7 @@ default boolean canPerformMetadataDelete(org.apache.hadoop.hive.ql.metadata.Tabl
     SearchArgument searchArgument) {
     return false;
   }
-  default List<FieldSchema> getPartitionKeys(org.apache.hadoop.hive.ql.metadata.Table hmsTable) {
+  default List<FieldSchema> getPartitionKeys(org.apache.hadoop.hive.ql.metadata.Table hmsTable, boolean latestSpecOnly) {
```

Comment:
please keep the original signature, we can't deprecate it immediately, just call the new parametrized method and pass 'true'